src/pages/PageNotFound/PageNotFound.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const PageNotFound = () => (
+  <>
```

Comment:
⛏️ We should be able to get rid of the react fragments.

Reply:
Addressed: d612efc22a699d66c1ab524fa76d86e4b6c65096